Terminal.Gui/ConsoleDrivers/V2/Logging.cs

```diff
@@ -65,4 +65,20 @@ public static void Trace (
         string className = Path.GetFileNameWithoutExtension (filePath);
         Logger.LogTrace ($"[{className}] [{caller}] {message}");
     }
+
+    /// <summary>
+    ///     Logs a warning message including the
```

Comment:
Just realised there's a hanging sentence here. Should finish "calling class and member names"

In both methods Warning and Trace